README.rst

```diff
@@ -148,10 +149,11 @@ If no specific bucket name is provided, the "default" bucket is used.
     client.update_bucket(id='payments', data={'description': 'My payments data.'})
 
     # Or modify some fields in an existing bucket.
-    client.patch_bucket(id='payments', data={'status': 'updated'})
+    # The Kinto server supports different types of patches, which can be used from kinto_http.patch_type.
+    client.patch_bucket(id='payments', data=BasicPatch({'status': 'updated'}))
 
     # It is also possible to manipulate bucket permissions (see later)
-    client.patch_bucket(id='payments', permissions={})
+    client.patch_bucket(id='payments', data=BasicPatch({'status': 'updated'}), permissions={})
```

Comment:
I wasn't sure how to update this, but the given example wouldn't work with the new code. There's no tests for it either. If we intend for it to work even when you don't pass data at all, we should add tests for it.